modules/apps/27-interchain-accounts/keeper/relay.go

```diff
@@ -15,7 +15,7 @@ import (
 
 // TODO: implement middleware functionality, this will allow us to use capabilities to
 // manage helper module access to owner addresses they do not have capabilities for
-func (k Keeper) TrySendTx(ctx sdk.Context, portID string, data interface{}) ([]byte, error) {
+func (k Keeper) TrySendTx(ctx sdk.Context, portID string, data interface{}, memo string) ([]byte, error) {
```

Comment:
should we take in the type as well so we don't have to change the api later?

Reply:
Great point. I guess right now it's just hardcoded to always be `EXECUTE`. I guess there is no way to interpret the type based on the data (which the more I think about it may be anything, not just a list of messages). 

I think it probably makes sense to let the caller pass in the type here. :+1: